src/transport/PASESession.cpp

```diff
@@ -65,7 +65,6 @@ void PASESession::Clear()
     memset(&mWS[0][0], 0, sizeof(mWS));
     memset(&mKe[0], 0, sizeof(mKe));
     mNextExpectedMsg = Protocols::SecureChannel::MsgType::PASE_Spake2pError;
-    mSpake2p.Init(nullptr);
```

Comment:
Ah, and the point is since we are initing `mSpake2p` with `mCommissioningHash` and we are clearing _that_ we don't need to mess with `mSpake2p`? This could actually use a comment explaining why we don't need to touch that member here.  Looks good to me with the comment added.